### PR TITLE
Move infrastructure and deploy files from stacks-dotnet-cqrs-events to stacks-dotnet-cqrs

### PR DESCRIPTION
**Pull request by underscoreHao**
[AB#4580](https://amido-dev.visualstudio.com/73884c9a-a68f-4f67-b2b5-b588c2eb8492/_workitems/edit/4580) Move infrastructure and deploy files from stacks-dotnet-cqrs-events to stacks-dotnet-cqrs

#### 📲 What

Move infrastructure and deploy files from `stacks-dotnet-cqrs-events` to `stacks-dotnet-cqrs`

#### 🤔 Why

We want to bring all infrastructure and pipeline setup from `cqrs-events` to this repository, so we can archive `cqrs-events`

#### 🛠 How

More in-depth discussion of the change or implementation.

#### 👀 Evidence

Screenshots / external resources / links / etc.
Link to documentation updated with changes impacted in the PR

#### 🕵️ How to test

Notes for QA

#### ✅ Acceptance criteria Checklist

- [ ] Code peer reviewed?
- [ ] Documentation has been updated to reflect the changes?
- [ ] Passing all automated tests, including a successful deployment?
- [ ] Passing any exploratory testing?
- [ ] Rebased/merged with latest changes from development and re-tested?
- [ ] Meeting the Coding Standards?

### CODE REVIEW

File: build/azDevOps/azure/api-contract-tests.yml [hunk 1]
```diff
@@ -57,6 +57,14 @@ stages:
           vmImage: 'ubuntu-latest'
         continueOnError: False
         steps:
+
+          - task: UseDotNet@2
+            displayName: 'Use .NET Core SDK 6.0.x'
+            inputs:
+              packageType: sdk
+              version: "6.0.x"
```
Comment (ElvenSpellmaker) [inline build/azDevOps/azure/api-contract-tests.yml:65]: Isn't there a variable for the version?

File: deploy/azure/app/eventhub/event_hub.tf [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    enabled  = true
+    encoding = "Avro"
+    destination {
+      name                = "EventHubArchive.AzureBlockBlob"
+      archive_name_format = "{Namespace}/{EventHub}/{PartitionId}/{Year}/{Month}/{Day}/{Hour}/{Minute}/{Second}"
+      blob_container_name = azurerm_storage_container.eh_storage_container.name
+      storage_account_id  = azurerm_storage_account.eh_storage.id
+    }
+  }
+}
```
Comment (ElvenSpellmaker) [inline deploy/azure/app/eventhub/event_hub.tf:39]: Newline please

File: deploy/azure/app/eventhub/vars.tf [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  type        = string
+  default     = "~4"
+  description = "Version of the Azure Function runtime to use"
+}
+
+variable "az_function_dotnet_version" {
+  type        = string
+  default     = "v6.0"
+  description = "Version of the .NET framework to use in the function"
+}
```
Comment (ElvenSpellmaker) [inline deploy/azure/app/eventhub/vars.tf:70]: Another newline please

File: build/azDevOps/azure/api-contract-tests.yml (hunk 1 above)
Comment (williamayerst) [reply build/azDevOps/azure/api-contract-tests.yml:65]: Contract test code hasn't changed, there's no variable for this currently.

File: build/azDevOps/azure/air-api.yml [hunk 4]
```diff
@@ -98,6 +100,7 @@ stages:
               artifact: tests
 
   - stage: Dev
+    # TODO: Temporary condition
```
[the remark this answers is not in the document]
Comment (williamayerst) [reply build/azDevOps/azure/air-api.yml:103]: Good spot - there are some deployment hiccups - when resolved this condition will need to be reinstated, thank you!

File: build/azDevOps/azure/api-contract-tests.yml (hunk 1 above)
Comment (ElvenSpellmaker) [reply build/azDevOps/azure/api-contract-tests.yml:65]: Ahh Contract tests in general aren't thought through well in Stacks. Noöne knows the ideal PACT flow and it's been poorly implemented currently.

**Review by underscoreHao — COMMENTED** [review]
Looks good

**Pull request merged**